package.json

```diff
@@ -7,23 +7,18 @@
   },
   "scripts": {
     "====== Generic===============": "==========================================",
-
-    "build:cpp-wasm-cmake":             "scripts/build-cmake.sh",
-    "build:cpp-wasm-cmake-clean":       "scripts/build-cmake.sh --clean",
-    "build:cpp-wasm-cmake-debug":       "scripts/build-cmake.sh --debug",
+    "build:cpp-wasm-cmake": "scripts/build-cmake.sh",
```

Comment:
We should decide whether we want to have the package.json formatter as I believe the line breaks were added intentionally (I noticed VSCode is happily formatting this file that's what I think happened here). Let's check on discord